plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/standard/StandardDeployJob.java

```diff
@@ -101,8 +85,10 @@ public IStatus runInWorkspace(IProgressMonitor monitor) throws CoreException {
 
       try {
         getJobManager().beginRule(config.getProject(), progress);
-        exporter.publish(config.getProject(), explodedWarDirectory, progress.newChild(10));
-        staging.stage(explodedWarDirectory, stagingDirectory, cloudSdk, progress.newChild(20));
+        new ExplodedWarPublisher().publish(
+            config.getProject(), explodedWarDirectory, progress.newChild(10));
+        new StandardProjectStaging().stage(
```

Comment:
whenever I see

new Foo(/* no args */).doSomething() 

that suggests to me doSomething() should be a static method, but that's for another PR